thirdparty/CMakeLists.txt

```diff
@@ -78,6 +78,14 @@ if(ENABLE_OV_CORE_UNIT_TESTS OR ENABLE_TESTS)
                                       TARGETS gmock gmock_main gtest gtest_main)
 endif()
 
+#
+# ZLib
+#
+if(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
+    # 4244 conversion from 'XXX' to 'YYY', possible loss of data
+    target_compile_options(zlib PRIVATE /wd4244)
+endif()
```

Comment:
can we move this to the dedicated file https://github.com/openvinotoolkit/openvino/blob/master/thirdparty/zlib/CMakeLists.txt ?